socket/socket.js

```diff
@@ -214,6 +239,7 @@ const insertMessages = async (msgData) => {
       fromUserId: msgData.fromUserId,
       toUserId: msgData.toUserId,
       content: msgData.content,
+      date: new Date()
```

Comment:
i think the isUnread field is supposed to be added here given it's "required" for a new message